wpilib/tests/test_shuffleboard.py

```diff
@@ -22,8 +30,7 @@ def test_enableActuatorWidgets(wpilib, getTable):
 def test_addPersistent(wpilib, getTable):
     tab = wpilib.shuffleboard.Shuffleboard.getTab("tacos")
     widget = tab.addPersistent("simplevalue", 1)
-    # k, how to kick networktables?
-    # assert widget.getEntry().isPersistent()
+    assert widget.getEntry().isPersistent()
```

Comment:
need help here, why is this failing?

Reply:
pynetworktables bug, oops...

Reply:
See https://github.com/robotpy/pynetworktables/pull/76